packages/fullstack/README.md

```diff
@@ -63,6 +63,81 @@ async fn get_meaning(of: String) -> Result<Option<u32>, ServerFnError> {
 }
 ```
 
+## Axum Integration
+
+If you have an existing Axum router or you need more control over the server, you can use the [`AxumRouterExt`](https://docs.rs/dioxus-fullstack/latest/dioxus_fullstack/prelude/trait.DioxusRouterExt.html) trait to integrate with your existing Axum router.
+
+First, make sure your `axum` dependency is optional and enabled by the server feature flag. Axum cannot be compiled to wasm, so if it is enabled by default, it will cause a compile error:
+
+```toml
+[dependencies]
+dioxus = { version = "*", features = ["fullstack"] }
+axum = { version = "0.7.0", optional = true }
+
+[features]
+server = ["dep:axum", "dioxus/server"]
+web = ["dioxus/web"]
+```
+
+Then we can set up dioxus with the axum server:
+
+```rust
+#![allow(non_snake_case)]
+use dioxus::prelude::*;
+
+// The entry point for the server
+#[cfg(feature = "server")]
+fn main() {
+    // Get the address the server should run on. If the CLI is running, the CLI proxies fullstack into the main address
+    // and we use the generated address the CLI gives us
+    let cli_args = dioxus_cli_config::RuntimeCLIArguments::from_cli();
+    let address = cli_args
+        .as_ref()
+        .map(|args| args.fullstack_address().address())
+        .unwrap_or_else(SocketAddr::from(([127, 0, 0, 1], 8080)));
+
+    // Set up the axum router
+    let serve_config = ServeConfig::new().unwrap();
```

Comment:
Maybe we shouldn't promo/demo the usage of unwrap here because of
1. Highly possible it is not what the users wants.
2. using `?` makes the code a little bit succinct